src/game-coordinator/services/server-configurator.service.ts

```diff
@@ -129,6 +161,7 @@ export class ServerConfiguratorService {
           slot.gameClass,
         );
         this.logger.debug(`[${game.gameServer.name}] ${cmd}`);
+        // skipcq: JS-0032
```

Comment:
?

Reply:
This is to skip DeepSource checks; it is angry at making for-loops with `await`s inside them (i.e. change it to `Promise.all()`), but in these cases I specifically want to await in each loop iteration.

Reply:
dumb